CoreDataAccessView/release/modules/ext/docs/CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/data-access-options.md

```diff
@@ -54,13 +54,27 @@ Each plugin that saves results will create one or more files in your
 save directory. It is up to each plugin how it names the files it
 creates.
 
+## Connection Logging
+
+The output of functionality responsible for connecting to data sources
+can be saved to constellation log files.
+
+<img src="../ext/docs/CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/resources/ConnectionLogging.png" alt="Connection Logging"/>
+
+When you select "Connection Logging" a timer will start that will disable 
+connection logging after 150 minutes. The time remaining is displayed on the 
```

Comment:
I wonder whether the sentence here could be reworded slightly? I think the sentence as it currently stands suggests that the connection logging is already enabled and selecting it will start the countdown to stop it, whereas I think the point you're wanting to get across here is that it's only enabled for a limited time (disabled after 150 minutes vs enabled for 150 minutes, if that makes sense)